.drone.yml

```
@@ -10,6 +10,9 @@ pipeline:
      - aws ecr get-login --region us-west-1 --no-include-email | sh
      - docker pull 995121555896.dkr.ecr.us-west-1.amazonaws.com/cloudlock/svc-statsd:latest
      - docker pull 995121555896.dkr.ecr.us-west-1.amazonaws.com/cloudlock/base-python3-hardened:latest
+     - aws ecr get-login --region us-east-1 --no-include-email --registry-ids 991726456769| sh
```

```suggestion
     - aws ecr get-login --region us-east-1 --no-include-email --registry-ids 991726456769 | sh
```
I noticed this space is missing, can it be parsed well without it?